src/shared/utils/cleanText.ts

```diff
@@ -13,22 +13,22 @@ import styles from './styles/clean-text.module.scss';
 // List of tags to remove from the library default accepted set
 const excludedTags = new Set(['a']);
 
-export const cleanTextDefaultOptions: IOptions = {
+export const cleanTextDefaultOptions = Object.freeze<IOptions>({
```

Comment:
We often want to freeze nested objects - shall we just go ahead and

Reply:
Agree, will tackle that in the follow up PR about entry pages though as I've made further modifications including Object.freeze there too.
Will add `deep-freeze` as a dependency and replaces all places using Object.freeze with it

Reply:
Handled in https://github.com/ebi-uniprot/uniprot-website/commit/82a1abde5fa648f057a6095d52c9d5c93854001d and https://github.com/ebi-uniprot/uniprot-website/commit/b06bf62e22320c865320bce6f83070007ec80869 (other branch)